README.md

```diff
@@ -175,6 +172,18 @@ as per [the official LLVM install guide](https://apt.llvm.org/).
 bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
 ```
 
+If the included test suite is being built and run, a GoogleTest library will be
+required. Here is an example of installing GoogleTest on a Debian-based Linux
+environment:
+
+```bash
+apt install libgtest-dev
```

Comment:
I guess google can't follow established practice?  A -dev install is normally when you want to modify the library, not just use it.  But apparently this is the correct way to install gtest, sadly.

Reply:
Don't worry. This is the normal Debian/Ubuntu/etc. naming convention. `libfoo-dev` contains headers, static libraries, and anything else needed in a build environment to build against the `foo` library. `libfoo` would contain runtime dependencies for the `foo` library, but when statically linking, you generally don't have any of those.

See:
https://packages.ubuntu.com/oracular/libgtest-dev

Also, like most Google-originating projects, the packaging is maintained separately from the original library. You can see the maintainers listed in the above link. They're a combination of Debian and Ubuntu contributors. They get all the credit and blame for the Debian packaging details, I suppose.